PyU4V/system.py

```diff
@@ -924,3 +925,28 @@ def get_ip_interface(self, director_id, port_id, interface_id,
             object_type=IP_INTERFACE, object_type_id=interface_id)
 
         return interface if interface else dict()
+
+    def change_local_user_password(
+            self, username, current_password, new_password):
+        """Function to allow users to change password for local user accounts.
+
+        Requires minimum version of Unisphere 9.2.1.x and API account must
+        have security admin role
+
+        Change local user password.
+        :param username: username for local account -- str
+        :param current_password: existing password for local account -- str
+        :param new_password: new password for local account -- str
+
```

Comment:
Gray area.  Should it return a dict from the modify_resource?

Reply:
:returns: -- dict

Reply:
There is actually no return on this, it will return None if successful